Sources/tart/Commands/Run.swift

```diff
@@ -37,9 +37,15 @@ struct Run: AsyncParsableCommand {
     let vmDir = try VMStorageLocal().open(name)
     vm = try VM(vmDir: vmDir)
 
+    var vncWrapper: VNCWrapper?
+
+    if vnc {
+      vncWrapper = VNCWrapper()
+    }
+
     Task {
       do {
-        try await vm!.run(recovery)
+        try await vm!.run(recovery, vncWrapper)
```

Comment:
Not sure of `VM` should "know" about VNC. Seems `VNCWrapper` only needs an instance of `VZVirtualMachine`... So `VNCWrapper` can potentially be initialized with `vm!.virtualMachine` and do something like

```swift
vncWrapper.start()
defer { vncWrapper.stop }
try await vm!.run(recovery)
```

Reply:
See https://github.com/cirruslabs/tart/pull/126/commits/b07cf7711f59045a1817f8c6368061ca2643e05e.